Introduction/1-image.md

```diff
@@ -10,21 +10,21 @@ In this approach, we use a combination of image processing and adb tool library
 
 We use ADB tool to take a screenshot of the phone screen and save it on the sdcard, and then use the next command to send over the image to the computer. Next, we use a set of image processing techniques to extract relevant features in the image that are required. Depending on the features, we can simuate the touch or swipe virtually again using adb tool. 
 
-We can run the above steps in a loop to automate a game that needs repition or has multiple levels.
+We can run the above steps in a loop to automate a game that needs repetition or has multiple levels.
 
 ### Advantages of this approach
-- We can get direct screenshots of our android phone because of which we get the accurate pixels values of the images.
+- We can get direct screenshots of our android phone screen and hence accurate pixel values.
 - Since we access the locations on the screen through their coordinates, we get accurate touch and swipe.
-- Circuits are not required.
+- Circuitry is not required.
```

Comment:
Please rephrase as `There's no need to use a circuit when we take this approach`